index.html

```diff
@@ -86,7 +86,63 @@ <h2>Travel</h2>
+                    <h3>Who can attend?</h2>
+                    <p>Any student who will be in high school, college, or graduate school on October 12, 2018 is eligible to apply.</p>
+                </div>
+                <div class="six columns">
+                    <h3>How much does it cost?</h2>
+                    <p>TotalityHacks is free&mdash;we will provide you with meals and drinks, so you don't have to spend a cent.</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <h3>Will travel reimbursements be provided?</h2>
+                    <p>When you get your acceptance email, you will also get information about travel. For some locations, we will send busses to a local university. For other locations, we establish a reasonable amount of money to spend on transportation (if you exceed it, we will still reimburse up to this amount).</p>
+                </div>
+                <div class="six columns">
+                    <h3>What should I bring?</h2>
+                    <p>
+                        You need to bring:
+                        <ul>
+                            <li>a laptop</li>
+                            <li>your cellphone</li>
+                            <li>chargers</li>
+                            <li>a sleeping bag</li>
+                            <li>toiletries</li>
+                            <li>a few changes of clothes</li>
+                        </ul>
+                    </p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <h3>How big are teams?</h2>
+                    <p>Teams can have up to four people. You don't need to have a team when you apply; there will be plenty of opportunities for you to find teammates, both before and during Totality.</p>
+                </div>
+                <div class="six columns">
+                    <h3>How do I find a team?</h2>
+                    <p>Once you're accepted, you can find a team online before the event or at the hackathon itself. If you have friends who are going to the event, you can also team up with them.</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="six columns">
+                    <h3>Is there a code of conduct?</h2>
+                    <p>Yes, there is. Nothing too out of the ordinary though. Generally, be nice to others. If you want the full guide, we are using the MLH code of conduct which can be found <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">here</a>.</p>
+                </div>
+                <div class="six columns">
+                    <h3>I have other questions.</h2>
+                    <p>You can contact us at <a href="mailto:hello@menlohacks.com">hello@menlohacks.com</a>, and we will do our best to answer your questions.</p>
```

Comment:
we don't want to email hello@menlohacks.com

Reply:
maybe I took a bit too much inspiration from MenloHacks...

Reply:
lol don't worry––this is what code review is for.